CommonData/column.cpp

```diff
@@ -709,6 +709,8 @@ bool Column::overwriteDataAndType(stringvec colData, columnType colType)
 	// In this case, the locale used is just UTF-8/C, so the locale chosen by the user should not be used to convert the values.
 	beginBatchedLabelsDB();
 	setValues(values, labels, 0, &changes, false);
+	if(computed && _codeType == computedColumnType::analysisNotComputed)
+		setCodeType(computedColumnType::analysis);
```

Comment:
And the other way around? Can `computed` be false and _codeType be `analysis`? Then the codeType should be set to `analysisNotComputed`

Reply:
I agree with you in principle, but the scenario would be:
a) a qmlcomponent (ComputedColumnField) makes a computed column
b) the R code then decides it isnt computed after all and should be not-computed... Seems a bit weird no?

I guess the computed argument here is more of a "hint", like, hey, init it as a computed.